OpenRA.Game/HotkeyModifiers.cs

```diff
@@ -0,0 +1,25 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2020 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+namespace OpenRA
+{
+	public class HotkeyModifiers
+	{
+		static Modifiers modifiers;
+		public static Modifiers GetModifierKeys() { return modifiers; }
+		internal static void HandleModifierKeys(Modifiers mods) { modifiers = mods; }
+
+		public static bool HasModifier(Modifiers mod)
```

Comment:
Can this be removed now?

Reply:
Imo it looks better to use `HotkeyModifiers.HasModifier` than `HotkeyModifiers.GetModifierKeys().HasFlag` but its only used 5 times in the code, so im open to remove it.